api/virtual_accounts/virtual_accounts.go

```diff
@@ -178,11 +236,29 @@ func (va *VAHandler) Create(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	fmt.Fprintf(w, "VA Number: %v Created!\n", vam.VaNum)
+	response := VAResponse{
+		Status:       1,
+		Notification: fmt.Sprintf("successfully create virtual account! virtual account number : %v", vam.VaNum),
+	}
+
+	// set response http status
+	w.WriteHeader(http.StatusCreated)
+
+	err = json.NewEncoder(w).Encode(response)
+	if err != nil {
+		helpers.HTTPError(w, http.StatusBadRequest, "unable to encode response")
+		return
+	}
 }
 
 // to edit VA
-func (va *VAHandler) Edit(w http.ResponseWriter, r *http.Request) {
+func (va *VAHandler) Update(w http.ResponseWriter, r *http.Request) {
+
+	// set response header
+	w.Header().Set("Content-Type", "application/json")
+
+	// get va number in url
+	vaNumber := chi.URLParam(r, "va_num")
```

Comment:
after receiving the va number, maybe it is better to check is the va number is valid or not, you can use the checkAccountVA function from file database_virtual_accounts.go